cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -30,6 +30,8 @@ import com.here.ort.model.OrtResult
 import com.here.ort.model.config.CopyrightGarbage
 import com.here.ort.model.config.OrtConfiguration
 import com.here.ort.model.config.Resolutions
+import com.here.ort.model.licenses.LicenseConfiguration
```

Comment:
Conceptually, shouldn't we ensure that the reporters use the same license configuration as the evaluator by adding the used configuration to the evaluator result?

Reply:
Having that guarentee seems nice. I prefer not doing so for the following reasons for now

* it is extra work to implement
* it is extra code to maintain and adds complexity
* it requires to re-run the evaluator for making visible local changes to that file.
  It was exactly the reason why all those override parameters have been added to those commands. 
  It makes working with things quicker imo.
* ensuring the same file is used in an environment where this is critical is still easy with the current implementation
* no additional dependency from reporter to evaluator
* I don't really see a strong need overall

Would you be fine with keeping this out ouf OrtResult ?

Reply:
Yes, but let's discuss offline what potential issues this could cause.